src/setup_node_env/babel_register/register.js

```diff
@@ -66,10 +66,11 @@ if (global.__BUILT_WITH_BABEL__) {
 
 // modifies all future calls to require() to automatically
 // compile the required source with babel
-require('babel-register')({
+require('@babel/register')({
   ignore,
   babelrc: false,
   presets: [
     require.resolve('@kbn/babel-preset/node_preset')
   ],
+  extensions: ['.es6', '.es', '.jsx', '.js', '.mjs', '.ts', '.tsx'],
```

Comment:
lots of extra extensions here, lets limit this to `'.js', '.ts', '.tsx'`

Reply:
👍